### PR TITLE
Added Bigtable emulator container support

### PR DESCRIPTION
**Pull request by saturnism**
/cc @eddumelendez

### CODE REVIEW

File: modules/gcloud/src/main/java/org/testcontainers/containers/BigtableEmulatorContainer.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    /**
+     * @return a <code>host:port</code> pair corresponding to the address on which the emulator is
+     * reachable from the test host machine. Directly usable as a parameter to the
+     * com.google.cloud.ServiceOptions.Builder#setHost(java.lang.String) method.
+     */
+    public String getEmulatorEndpoint() {
+        return getContainerIpAddress() + ":" + getMappedPort(PORT);
+    }
+
+    public String getEmulatorHost() {
```
Comment (bsideup) [inline modules/gcloud/src/main/java/org/testcontainers/containers/BigtableEmulatorContainer.java:41]: given that we have `getHost()` now, do you think we still need this one?

File: modules/gcloud/src/main/java/org/testcontainers/containers/BigtableEmulatorContainer.java [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        withCommand("/bin/sh", "-c", CMD);
+    }
+
+    /**
+     * @return a <code>host:port</code> pair corresponding to the address on which the emulator is
+     * reachable from the test host machine. Directly usable as a parameter to the
+     * com.google.cloud.ServiceOptions.Builder#setHost(java.lang.String) method.
+     */
+    public String getEmulatorEndpoint() {
+        return getContainerIpAddress() + ":" + getMappedPort(PORT);
```
Comment (bsideup) [inline modules/gcloud/src/main/java/org/testcontainers/containers/BigtableEmulatorContainer.java:38]:
```suggestion
        return getHost() + ":" + getMappedPort(PORT);
```

File: modules/gcloud/src/main/java/org/testcontainers/containers/BigtableEmulatorContainer.java (hunk 1 above)
Comment (saturnism) [reply modules/gcloud/src/main/java/org/testcontainers/containers/BigtableEmulatorContainer.java:41]: will remove.

Comment (saturnism) [reply modules/gcloud/src/main/java/org/testcontainers/containers/BigtableEmulatorContainer.java]: removed.

**Review by bsideup — APPROVED** [review]

**Pull request merged**